helix-term/src/commands.rs

```diff
@@ -1000,31 +1000,66 @@ fn goto_file_vsplit(cx: &mut Context) {
     goto_file_impl(cx, Action::VerticalSplit);
 }
 
+fn trim_and_resolve_path(
+    selection_text: String,
```

Comment:
This should accept &str, then you don't need to to_string() in the call below which allocates